lib/src/component_declaration/flux_component.dart

```diff
@@ -82,6 +83,7 @@ abstract class FluxUiStatefulComponent<TProps extends FluxUiProps, TState extend
 ///
 /// Private so it will only get used in this file, since having lifecycle methods in a mixin is risky.
 abstract class _FluxComponentMixin<TProps extends FluxUiProps> implements BatchedRedraws {
+  final Logger _logger = new Logger('_FluxComponentMixin');
```

Comment:
There's not clear guidance that I can find, but I think we should [clearListeners](https://github.com/dart-lang/logging/blob/master/lib/logging.dart#L135) on the `Logger` when things are disposed.